content/blog/esc-imports-visualization/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Pulumi ESC: Visualizing Environment Imports"
```

Comment:
"Visualization" makes me think about a graph or chart.  I would focus on the enhanced transparency.
```suggestion
title: "Pulumi ESC: Enhancing Transparency for Environment Imports"
```

Reply:
Would have to fix throughout the doc

Reply:
Updated in the latest commit